src/Storage/Storage/Blob/Cmdlet/GetAzureStorageBlobContent.cs

```diff
@@ -433,6 +433,38 @@ internal string GetFullReceiveFilePath(string fileName, string blobName, DateTim
             return filePath;
         }
 
+        /// <summary>
+        /// When request doesn't container a proper bearer token, server will return 401 error include the audience of the required bearer token.
+        /// This function will get the audience of bearer token from SDK exception message.
+        /// If server not return audience, will output null.
+        /// </summary>
+        private string GetAudienceFrom401ExceptionMessage(global::Azure.RequestFailedException exception)
+        {
+            string authenticateHeaderName = "WWW-Authenticate";
+            string audience = null;
+            foreach (var header in exception.GetRawResponse().Headers)
+            {
+                string headerName = header.Name;
+                if (headerName.StartsWith(authenticateHeaderName))
+                {
+                    string headerValue = header.Value;
+                    WriteDebugLog(string.Format("Found header: {0}", headerValue));
```

Comment:
It's better to also add the header name.

Reply:
Added